### PR TITLE
Eperez enum security

### PR DESCRIPTION
**Pull request by enriquepablo**
Gather translatable messages for the security app in an enum

### CODE REVIEW

File: src/eduid_webapp/security/views/webauthn.py [hunk 1]
```diff
@@ -135,17 +139,20 @@ def remove(user, credential_key):
     security_user = SecurityUser.from_user(user, current_app.private_userdb)
     tokens = security_user.credentials.filter(FidoCredential)
     if tokens.count <= 1:
-        return {'_error': True, 'message': 'security.webauthn-noremove-last'}
+        return {'_error': True, 'message': str(SecurityMsg.no_last.value)}
```
Comment (fredrikt) [inline src/eduid_webapp/security/views/webauthn.py:142]: unnecessary str()

File: src/eduid_webapp/security/helpers.py [hunk 2]
```diff
@@ -23,6 +25,52 @@
...
+    # success adding a new nin
+    add_success = 'nins.successfully_added'
+    # The user tried to register more than the allowed number of tokens
+    max_tokens = 'security.u2f.max_allowed_tokens'
+    max_webauthn = 'security.webauthn.max_allowed_tokens'
+    # missing u2f enrollment data
+    missing_data = 'security.u2f.missing_enrollment_data'
+    # successfully registered u2f token
+    u2f_registered = 'security.u2f_register_success'
+    # No u2f tokens forund for the user
```
Comment (fredrikt) [inline src/eduid_webapp/security/helpers.py:52]: forund -> found

File: src/eduid_webapp/security/views/webauthn.py (hunk 1 above)
Comment (enriquepablo) [reply src/eduid_webapp/security/views/webauthn.py]: removed

File: src/eduid_webapp/security/helpers.py (hunk 2 above)
Comment (enriquepablo) [reply src/eduid_webapp/security/helpers.py]: corrected

**Pull request merged**